workbench/src/Components/LandingPage/LoadDesign/LoadDesign.scss

```
@@ -0,0 +1,169 @@
+
+    .create-design-row {
+        display:flex;
+        flex-direction:row;
+        padding: 0 20px;
+        align-items: center;
+        margin-bottom: 10px;
+
+        .file-name-input {
+            flex:1;
+            font-size: 14px;
+            border: none;
```

Stylelint (17.6.0)

[error] Unexpected duplicate "border" (declaration-block-no-duplicate-properties)

(declaration-block-no-duplicate-properties)